int8 state_driver
int8 state_mes
int8 state_error

